content/docs/cucumber/configuration.md

```diff
@@ -293,7 +334,9 @@ output.
 
 ## Default Profile
 {{% block "java" %}}
-Profiles are not available in Java.
+Profiles are not available in Java.  See above.
```

Comment:
Same comment as above regarding "Cucumber" profiles vs Maven profiles as we know them in Java

Reply:
Fixed :)